### PR TITLE
Modsource 48

### PR DESCRIPTION
**Pull request by Anatolii-Starkov**
Added indexes for source record view in the records, raw_records, marc_records tables

### CODE REVIEW

File: mod-source-record-storage-server/src/main/resources/templates/db_scripts/create_records_view.sql [hunk 1]
```diff
@@ -59,3 +59,10 @@ CREATE OR REPLACE VIEW source_records_view AS
      JOIN raw_records ON records.jsonb->>'rawRecordId' = raw_records.jsonb->>'id'
      LEFT JOIN marc_records ON records.jsonb->>'parsedRecordId' = marc_records.jsonb->>'id'
      WHERE records.jsonb->>'parsedRecordId' IS NOT NULL;
+
+
+CREATE INDEX idx_btree_raw_record_id ON records USING BTREE ((jsonb ->> 'rawRecordId'));
+CREATE INDEX idx_btree_parsed_record_id ON records USING BTREE ((jsonb ->> 'parsedRecordId'));
+
+CREATE INDEX idx_btree_raw_records_id ON raw_records USING BTREE ((jsonb ->> 'id'));
```
Comment (OleksiiKuzminov) [inline mod-source-record-storage-server/src/main/resources/templates/db_scripts/create_records_view.sql]: I think we need add for errorRecords index also

File: mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -93,6 +93,18 @@
...
+      ],
+      "foreignKeys": [
+        {
+          "fieldName": "rawRecordId",
+          "targetTable": "raw_records",
+          "tOps": "ADD"
+        },
+        {
+          "fieldName": "parsedRecordId",
+          "targetTable": "marc_records",
```
Comment (KaterynaSenchenko) [inline mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json]: parsedRecord can be any type, not just MARC, therefore they will be stored in different tables. It is incorrect to put a foreignKey constraint on this field referencing marc_records table

Comment (Anatolii-Starkov) [reply mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json]: Please talk it over with @TarasSpashchenko. If it's bad approach the you will able to revert last commit and use Btree indexes implementation of that I had already pushed before.

Comment (Igor-Gorchakov) [reply mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json]: @Anatolii-Starkov , we've already discussed on Friday, using you changes made before.

Comment (KaterynaSenchenko) [reply mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json]: Reverted last commit. We will stick with btrees for now.

**Pull request merged**